lib/assessment.js

```diff
@@ -4,24 +4,31 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(arg) {
+  min = math.min();
```

Comment:
A few notes:

* Assigning to `min` like this will replace the reference to this function with whatever is on the right of the equal sign. The first time you run it, it will overwrite the `min` variable. The next time you invoke it, `min` will refer to something else.

* Identifiers (e.g., variable names) are case-sensitive. `Math` and `math` are different variables.

* This function was intended to have an empty parameter list. This lesson introduced the `arguments` pseudo-array, available only inside functions, which contains the argument values passed to the function when it was invoked.

* Finally, using the built in methods on the `Math` object is cheating! You were supposed to write it yourself!